src/components/ChatEntry.css

```diff
@@ -97,4 +97,12 @@ button {
 
 .chat-entry.remote .entry-bubble:hover::before {
   background-color: #a9f6f6;
+}
+
+.full-heart {
+  content: '❤️'
+}
+
+.empty-heart {
+  content: '🤍'
```

Comment:
These heart css blocks don't look like they're being used. Try to remember to clean up unused code before finalizing a project! This improves the maintainability of a project. 🧹